ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -1,25 +1,42 @@
-{% if aaa_authorization is defined and aaa_authorization is not none %}
+{% if aaa_authorization is arista.avd.defined %}
 
 ## AAA Authorization
 
 ### AAA Authorization Summary
 
 | Type | User Stores |
 | ---- | ----------- |
-{%     if aaa_authorization.exec.default is defined %}
+{%     if aaa_authorization.exec.default is arista.avd.defined %}
 | Exec | {{ aaa_authorization.exec.default }} |
 {%     endif %}
+{%     if aaa_authorization.config_commands is arista.avd.defined(true) %}
 
-{%     if aaa_authorization.config_commands is defined and aaa_authorization.config_commands == true %}
 Authorization for configuration commands is enabled.
 {%     else %}
+
 Authorization for configuration commands is disabled.
 {%     endif %}
+{%     if aaa_authorization.serial_console is arista.avd.defined(true) %}
 
-{%     if aaa_authorization.serial_console is defined and aaa_authorization.serial_console == true %}
 Authorization for serial console is enabled.
+{%     endif %}
+{%     if aaa_authorization.commands.privilege is arista.avd.defined or aaa_authorization.commands.all_default is arista.avd.defined %}
+
+### AAA Authorization Privilege Levels Summary
 
+| Privilege Level | Group |
+| --------------  | ----- |
+{%         if aaa_authorization.commands.all_default is arista.avd.defined %}
+| all | {{ aaa_authorization.commands.all_default }} |
+{%         endif %}
+ {%        for command_level in aaa_authorization.commands.privilege | arista.avd.natural_sort %}
```

Comment:
I just found in Line 32, there is an empty space before ```{%```, this is causing MD errors. Please remove that empty space.